include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -30,7 +30,7 @@
  *
  * This value is used by some API functions to behave as this version of the header expects.
  */
-#define ORT_API_VERSION 14
+#define ORT_API_VERSION 15
```

Comment:
You need to update all the other places this is referenced, like the comments in the OrtApi structure and the name of it.